src/utils/jwt.ts

```diff
@@ -29,7 +29,8 @@ export const getUserJwt = (
   privateKey: NonEmptyString,
   tokenUser: TokenUser | TokenUserL2,
   tokenTtlSeconds: NonNegativeInteger,
-  issuer: NonEmptyString
+  issuer: NonEmptyString,
+  keyid?: NonEmptyString
```

Comment:
Should be better if the value of keyId is binded with the publicKey/privateKey pair (public key fingerprint?). This will prevent errors when a key rotation happens if someone forget to update the value accordingly.

Reply:
right @BurnedMarshal , the publicKey/privateKey used to sign/verify JWT are generated by terraform and in the same moment, using the public key we generate the relative kid. Good point for use the same way on rotation.

Reply:
If this value is generated by terraform any configuration update executed outside of Terraform can broke the application. We have at the application level all we need with only the `privateKey`. I think that is more safe if the value of the KID params is extracted from there at application level.